gobblin-utility/src/main/java/gobblin/util/HadoopUtils.java

```diff
@@ -110,6 +113,7 @@ public static Configuration newConfiguration() {
     // Add a new custom filesystem mapping
     conf.set("fs.sftp.impl", "gobblin.source.extractor.extract.sftp.SftpLightWeightFileSystem");
     conf.set("fs.sftp.impl.disable.cache", "true");
+    conf.set("fs.adl.impl", "org.apache.hadoop.fs.adl.AdlFileSystem");
```

Comment:
Will add a ADL module as suggested. I am having trouble finding their github page. It's not part of hadoop project. Let me either do the PR or create an issue on this.